autoscaler/migration.go

```diff
@@ -50,3 +50,42 @@ func MigrateCollectorsWorkloadToNewLabels(ctx context.Context, c client.Client,
 	return nil
 
 }
+func OldVerKubernetesMigrateCollectorsWorkloadToNewLabels(ctx context.Context, c client.Client, ns string) error {
```

Comment:
The purpose of this function is to migrage pre 91 versions due to some breaking changes.

Since no one can currently sun version `v1.0.91` with old k8s versions, I think this functionallity can simply be skipped for old versions (and later removed for all versions once we deprecate `v1.0.91`